libethereum/Executive.cpp

```diff
@@ -109,12 +109,14 @@ void Executive::initialize(Transaction const& _transaction)
             m_excepted = TransactionException::InvalidSignature;
             throw;
         }
-        if (m_t.nonce() != nonceReq)
+        if (m_t.nonce() < nonceReq)
```

Comment:
I believe this is going to break consensus, allowing the blocks with transactions in incorrect order, so this won't do.

Reply:
@winsvega do we have any blockchain tests with blocks containing transactions with invalid nonces?

Reply:
@gumb0 Ah you're correct, I had thought that the `Executive `was only invoked when submitting txs but it looks like it's invoked when executing txs as a part of importing txs into a block: `Block::sync->Block::execute->State::execute->State::executeTransaction->Executive::initialize`

...and when importing blocks into the chain: 

`BlockChain::sync -> BlockChain::import -> Block::enactOn -> Block::enact -> Block::execute -> State::execute -> State::executeTransaction -> Executive::initialize`

We definitely don't want to change the nonce check in the Executive since that's consensus-critical code. Thanks for catching this.